mem/mem.go

```diff
@@ -139,35 +168,59 @@ func getStats(stats map[string]interface{}) error {
 		if err != nil {
 			return err
 		}
-
 		switch name {
 		case "MemFree", "Buffers", "Cached", "Slab":
 			memSum += value * unit
 		}
 
-		tmpStats[name] = value * unit
+		tmpStats[formatName(name)] = value * unit
 	}
 
-	if tmpStats["MemTotal"] < memSum {
+	if tmpStats["mem_total"].(uint64) < memSum {
 		return fmt.Errorf("More value mismatch! More used then available")
 	}
 
-	tmpStats["MemUsed"] = tmpStats["MemTotal"] - memSum
-
-	total := tmpStats["MemUsed"] + memSum
+	tmpStats["mem_used"] = tmpStats["mem_total"].(uint64) - memSum
+	total := tmpStats["mem_used"].(uint64) + memSum
```

Comment:
Why use an interface instead of uint64 if you're type asserting? It seems safer to not do that.